docs/team/jaychenyj.md

```diff
@@ -65,7 +85,12 @@ Given below are my contributions to the project.
       , [\#98](https://github.com/AY2021S2-CS2103T-W10-2/tp/pull/98)
       , [\#100](https://github.com/AY2021S2-CS2103T-W10-2/tp/pull/100)
       , [\#101](https://github.com/AY2021S2-CS2103T-W10-2/tp/pull/101)
-    * Contributed to forum discussions:
+      , [\#113](https://github.com/AY2021S2-CS2103T-W10-2/tp/pull/113)
+      , [\#124](https://github.com/AY2021S2-CS2103T-W10-2/tp/pull/124)
+      , [\#125](https://github.com/AY2021S2-CS2103T-W10-2/tp/pull/125)
+      , [\#159](https://github.com/AY2021S2-CS2103T-W10-2/tp/pull/159)
+      , [\#168](https://github.com/AY2021S2-CS2103T-W10-2/tp/pull/168)
+      , [\#174](https://github.com/AY2021S2-CS2103T-W10-2/tp/pull/174)
     * Reported bugs and suggestions for other teams in the class:
     * Some parts of the history feature I added was adopted by several other classmates:
```

Comment:
Maybe you could remove it if there's nothing to include

Reply:
Will be adding for the team I reviewed for in the next iteration :)